rest/route/annotations_test.go

```diff
@@ -344,8 +344,13 @@ func TestAnnotationByTaskPutHandlerParse(t *testing.T) {
 	assert.NoError(t, db.ClearCollections(annotations.Collection, task.Collection, task.OldCollection))
 	tasks := []task.Task{
 		{Id: "t1", Execution: 1, Status: evergreen.TaskFailed},
-		{Id: "t2", Execution: 1, Status: evergreen.TaskFailed},
-		{Id: "t3", Execution: 1, Status: evergreen.TaskSucceeded},
+		{Id: "t2", Execution: 1, Status: evergreen.TaskSystemUnresponse},
+		{Id: "t3", Execution: 0, Status: evergreen.TaskSystemFailed},
+		{Id: "t4", Execution: 0, Status: evergreen.TaskTimedOut},
+		{Id: "t5", Execution: 0, Status: evergreen.TaskSetupFailed},
+		{Id: "t6", Execution: 0, Status: evergreen.TaskSucceeded},
+		{Id: "t7", Execution: 0, Status: evergreen.TaskWillRun},
+		{Id: "t8", Execution: 0, Status: evergreen.TaskAborted},
```

Comment:
I _think_ that TaskAborted is only used on the UI, so it wouldn't be relevant for this because this deals with the db directly.